test/controllers/vendor_records_controller_test.rb

```diff
@@ -98,6 +98,29 @@ class VendorsRecordsControllerTest < ActionController::TestCase
     end
   end
 
+  test 'should validate vendor patients' do
+    for_each_logged_in_user([ADMIN]) do
+      # Clear out old vendor patients (loaded from factories)
+      @vendor.fhir_patient_bundles.destroy
+
+      filename = Rails.root.join('test', 'fixtures', 'artifacts', 'good_patient_upload.zip')
```

Comment:
minor comment that 'good_patient_upload.zip' is a confusing name for a zip with patients with validation errors.